compiler/main.go

```diff
@@ -9,11 +9,9 @@ import (
 	compiler "github.com/maratona-run-time/Maratona-Runtime/compiler/src"
 	"github.com/maratona-run-time/Maratona-Runtime/utils"
 	"github.com/martini-contrib/binding"
+	"github.com/rs/zerolog"
 
 	"os"
```

Comment:
I think we are importing non github packages first on the rest of the codebase, but this is just a detail